src/InputField/InputField.stories.mdx

```diff
@@ -0,0 +1,99 @@
+import { useState } from 'react';
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks';
+import InputField from '../InputField';
+import Input from '../Input';
+
+<Meta title="Elements|InputField" component={InputField} />
+
+# Input Field
+
+### Basic Input Field component
+
+```js
+import { InputField } from 'minerva-ui';
+import { Input } from 'minerva-ui"
+```
+
+```jsx
+() => {
+  const [name, setName] = React.useState('')
+  const [error, setError] = React.useState('')
+
+  return (<InputField
+            label="Label"
```

Comment:
If the formatting for `.mdx` files is giving you any trouble, you should be able to use this extension:

https://marketplace.visualstudio.com/items?itemName=silvenon.mdx